packages/rest-typings/src/v1/emojiCustom.ts

```diff
@@ -52,7 +52,7 @@ export const isEmojiCustomList = ajv.compile<emojiCustomList>(emojiCustomListSch
 
 export type EmojiCustomEndpoints = {
 	'/v1/emoji-custom.all': {
-		GET: (params: PaginatedRequest<{ query: string }, 'name'>) => PaginatedResult<{
+		GET: (params: PaginatedRequest<{ name?: string }, 'name'>) => PaginatedResult<{
```

Comment:
you said you're not removing `query`

Reply:
If you look at PaginatedRequest, you will see already have an optional query there. There is no necessity of adding it as a generic type in  PaginatedRequest